src/calculation/HTMLBuilder.ts

```diff
@@ -317,5 +379,18 @@ export function calculateClauseCoverage(relevantStatements: StatementResult[], c
     allRelevantClauses.filter(clause => clause.final === FinalResult.TRUE),
     (c1, c2) => c1.libraryName === c2.libraryName && c1.localId === c2.localId
   );
-  return ((coveredClauses.length / allUniqueClauses.length) * 100).toPrecision(3);
+
+  const uncoveredClauses = allUniqueClauses.filter(c => {
+    if (coveredClauses.find(coveredC => c.libraryName === coveredC.libraryName && c.localId === coveredC.localId)) {
```

Comment:
suggest simplify this if/else boolean
```suggestion
    return !(coveredClauses.find(coveredC => c.libraryName === coveredC.libraryName && c.localId === coveredC.localId)) {
```

Reply:
Good call! Implemented this in f4b34e6.